library/core/networking/src/main/java/org/quiltmc/qsl/networking/api/EntityTrackingEvents.java

```diff
@@ -30,13 +30,43 @@ public final class EntityTrackingEvents {
 	 * An event that is called before player starts tracking an entity.
 	 * Typically, this occurs when an entity enters a client's view distance.
 	 * This event is called before the player's client is sent the entity's {@link Entity#createSpawnPacket() spawn packet}.
+	 *
+	 * @apiNote Since the client will not know about the entity at this point, you probably don't want to send packets
+	 * referencing the entity here. Do that in {@link #AFTER_START_TRACKING} instead.
 	 */
-	public static final Event<StartTracking> START_TRACKING = Event.create(StartTracking.class, callbacks -> (trackedEntity, player) -> {
+	public static final Event<StartTracking> BEFORE_START_TRACKING = Event.create(StartTracking.class, callbacks -> (trackedEntity, player) -> {
 		for (StartTracking callback : callbacks) {
 			callback.onStartTracking(trackedEntity, player);
 		}
 	});
 
+	/**
+	 * An event that is called before player starts tracking an entity.
+	 * Typically, this occurs when an entity enters a client's view distance.
+	 * This event is called before the player's client is sent the entity's {@link Entity#createSpawnPacket() spawn packet}.
+	 *
+	 * @apiNote Since the client will not know about the entity at this point, you probably don't want to send packets
+	 * referencing the entity here. Do that in {@link #AFTER_START_TRACKING} instead.
+	 *
+	 * @deprecated Use {@link #BEFORE_START_TRACKING} instead. It's the same event with a more descriptive name.
```

Comment:
```suggestion
	 * @deprecated Renamed to {@link #BEFORE_START_TRACKING}.
```
...maybe? I feel like the original sentence is too wordy.

Reply:
Yeah, that's much better.